package-lock.json

```diff
@@ -1925,6 +1925,43 @@
         "node": ">=14"
       }
     },
+    "node_modules/@pnpm/config.env-replace": {
```

Comment:
Please rerun `npm install` on this repo. I'm getting a small change to package lock locally when I did